src/traces/contourgl/convert.js

```diff
@@ -77,7 +77,7 @@ proto.handlePick = function(pickResult) {
         ],
         textLabel: this.textLabels[index],
         name: this.name,
-        pointIndex: [xIndex, yIndex],
+        pointIndex: [yIndex, xIndex],
```

Comment:
thanks @dfcreative for that https://github.com/plotly/plotly.js/issues/1825#issuecomment-311787182